docs/core/tools/dotnet-build.md

```diff
@@ -26,9 +26,17 @@ dotnet build [-h|--help]
 
 ## Description
 
-The `dotnet build` command builds the project and its dependencies into a set of binaries. The binaries include the project's code in Intermediate Language (IL) files with a *.dll* extension and symbol files used for debugging with a *.pdb* extension. A dependencies JSON file (*.deps.json*) is produced that lists the dependencies of the application. A *.runtimeconfig.json* file is produced, which specifies the shared runtime and its version for the application.
+The `dotnet build` command builds the project and its dependencies into a set of binaries. The binaries include the project's code in Intermediate Language (IL) files with a *.dll* extension.  Depending on the project type and settings, other files may be included, such as:
```

Comment:
> Depending on the project type and settings...

Any details on the type and settings?

I miss the `<EnableDynamicLoading>` property which makes libraries produce runtime configuration files and copy Nuget dependencies.

Reply:
We do want to add more documentation on the various project settings, but it wouldn't be on this page.